.azure-pipelines-templates/matrix.yml

```diff
@@ -12,6 +12,8 @@ parameters:
     SGX:
       container: sgx
       pool: 1es-dcv2-focal
+    SNP:
+      pool: sev-snp-pool
```

Comment:
Could you add instructions on how to create and configure this pool, even if this eventually subsumed by #4190, in our internal doc repo please?

Reply:
https://dev.azure.com/MSRC-CCF/InternalDoc/_wiki/wikis/Internal%20Doc/8/Agent-Pools